dask/dask.yaml

```diff
@@ -8,7 +8,9 @@ tokenize:
 
 dataframe:
   backend: "pandas"  # Backend dataframe library for input IO and data creation
-  shuffle-compression: null  # compression for on disk-shuffling. Partd supports ZLib, BZ2, SNAPPY
+  shuffle:
+    algorithm: null
+    compression: null  # compression for on disk-shuffling. Partd supports ZLib, BZ2, SNAPPY
```

Comment:
If it's only about disk-based shuffling, what about `dataframe.shuffle.disk.compression`?

Reply:
I could see this being used elsewhere and think it's fine for a specific algorithm to ignore such a value and suggest to keep it as is